meinberlin/apps/dashboard/views.py

```diff
@@ -49,8 +52,9 @@ def post(self, request, *args, **kwargs):
         project = self.get_object()
         module = module_models.Module(
             name=self.blueprint.title,
-            weight=len(project.modules) + 1,
+            weight=max([module.weight for module in project.modules]) + 1,
```

Comment:
could you also work with a value_list here? As there won't be a lot of modules, using a for loop should not be a problem, but if possible, I guess, it is a bit better to try to use DB functionalities.